Bindings/bindings.json

```diff
@@ -113,6 +113,26 @@
                     "label": "$eventHubTrigger_connection_label",
                     "help": "$eventHubTrigger_connection_help",
                     "placeholder": "[variables('selectConnection')]"
+                },
+                {
+                    "name": "cardinality",
+                    "value": "enum",
+                    "resource": "EventHub",
+                    "required": false,
+                    "defaultValue": "many",
+                    "enum": [
+                        {
+                            "value": "one",
+                            "display": "one"
+                        },
+                        {
+                            "value": "many",
+                            "display": "many"
+                        }
+                    ],
+                    "label": "$eventHubTrigger_cardinality_label",
```

Comment:
https://github.com/Azure/azure-webjobs-sdk-templates/pull/430/files?w=1#diff-73189aaa4df7ca5d944ae99dd2d5949aR1149

Didn't show up in the diff, funky.